test/test_autograd.py

```diff
@@ -2865,26 +2865,6 @@ def test_var_mean_differentiable(self):
         torch.autograd.backward(r2, grad)
         self.assertTrue(torch.allclose(input1.grad, input2.grad, rtol=0.01, atol=0.0))
 
-    @skipIfNoLapack
```

Comment:
There are another test https://github.com/pytorch/pytorch/blob/f71a0daeb747148520ef275fa65b79cc6f5fa36b/test/test_autograd.py#L2890-L2947 that can be migrated as well.

Reply:
I also thought so when doing a quick search, but it turns out that that's from `torch.lobpcg`!

Reply:
ah okay!